silk-workspace/src/main/scala/org/silkframework/workspace/activity/workflow/LocalWorkflowExecutor.scala

```diff
@@ -168,7 +174,9 @@ case class LocalWorkflowExecutor(workflowTask: ProjectTask[Workflow],
 
       if (!cancelled && !isExecutedWorkflow) {
         try {
-          executeAndClose("Executing", operatorNode.nodeId, operatorTask, inputResults.flatten, executorOutput) { result =>
+          val registry = MeterRegistryProvider.meterRegistry
+          val stopwatch: Timer.Sample = Timer.start()
+          val result = executeAndClose("Executing", operatorNode.nodeId, operatorTask, inputResults.flatten, executorOutput) { result =>
```

Comment:
This would not include the execution time of failed workflows. That's probably fine, but we should ask for feedback if this is expected.